Library/Homebrew/resource.rb

```diff
@@ -10,7 +10,7 @@ class Resource
 
   attr_reader :mirrors, :specs, :using
   attr_writer :version
-  attr_accessor :download_strategy, :checksum
+  attr_accessor :download_strategy, :checksum, :extracts
```

Comment:
The name here doesn't seem to fit for me but I'm struggling to think of a better one. Any ideas?